test/context.c

```diff
@@ -29,22 +29,19 @@ typedef struct _MyItem {
 bool CheckMemory(MyContext * pContext)
 {
 	MyItem * p;
-	int i;
-
 	//  Walk memory and check every block
 
 	for (p =  (MyItem *) pContext->pFirst; p != NULL; p = p->pNext) {
 		if (p->pad[0] == (byte) 0xab) {
 			//  Block has been freed
-			for (i = 0; i < p->size + 8; i++) {
+			for (unsigned i = 0; i < p->size + 8; i++) {
 				if (p->pad[i] != (byte) 0xab) {
 					fprintf(stderr, "Freed block is modified");
 					assert(false);
 				}
 			}
-		}
-		else if (p->pad[0] == (byte) 0xef) {
-			for (i = 0; i < 4; i++) {
+		} else if (p->pad[0] == (byte) 0xef) {
```

Comment:
Did clang-format miss this case?  "}" and "else" on separate lines

Reply:
this MR still has not clang-format, the MR has not landed yet